source/features/deep-reblame.tsx

```diff
@@ -68,7 +73,9 @@ async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchor
 
 	blameElement.blur(); // Hide tooltip after click, it’s shown on :focus
 
-	const prBlameCommit = await getPullRequestBlameCommit(prCommit, prNumber);
+	const currentFilename = getCleanPathname().split('/').slice(-3).join('/');
```

Comment:
This fails on branches with slashes. Try with `getReference` (as used in other features)

Maybe just:
```suggestion
	const [, currentFilename] = getCleanPathname().split(getReference());
```